src/test/java/org/broadinstitute/hellbender/utils/plotter/AlleleFractionSegmentedPlotterTest.java

```diff
@@ -8,7 +8,7 @@
 import java.io.File;
 import java.io.IOException;
 
-public class AlleleFractionPlotterTest extends BaseTest {
+public class AlleleFractionSegmentedPlotterTest extends BaseTest {
     @Test
     public void testAlleleFractionPlotting() throws IOException {
```

Comment:
I do not like tests that throw IOExceptions.  I would argue that you should catch the IOException in the test and use:

```java
try {
    //.....snip....
} catch (final IOException ioe) {
    Assert.fail("Could not run test due to problems with a file", ioe);
}
```